src/hearings/containers/hearing-actuals/hearing-actuals.component.ts

```diff
@@ -1,15 +1,24 @@
-import {Component} from '@angular/core';
-import {ACTION} from '../../models/hearings.enum';
-import {HearingsService} from '../../services/hearings.service';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { Subscription } from 'rxjs';
+import { ACTION } from '../../models/hearings.enum';
+import { HearingsService } from '../../services/hearings.service';
+import * as actions from '../../store/actions/hearing-actuals.action';
 
 @Component({
   selector: 'exui-hearing-actuals',
   templateUrl: './hearing-actuals.component.html',
   styleUrls: ['./hearing-actuals.component.scss']
 })
-export class HearingActualsComponent {
+export class HearingActualsComponent implements OnInit, OnDestroy {
 
-  constructor(private readonly hearingsService: HearingsService) {
+  private sub: Subscription;
+
+  constructor(private readonly hearingsService: HearingsService, private store: Store<any>) {
+  }
+
+  public ngOnInit(): void {
+    this.store.dispatch(new actions.GetHearingActuals('1'));
```

Comment:
Could this hearingID not be hard coded? You could pass it to the route like hearings/actuals/[hearingId] when click the 'Add or edit' button, and get the hearing ID from the route.

Reply:
I was a bit confused how the routes would work. This should have been covered with Paddy's ticket. The actual route should be something like `/hearings/:id/actuals/hearing-timings`